external/radiation/python/radiation_driver.py

```diff
@@ -747,31 +747,9 @@ def _GFS_radiation_driver(
                     if ccnd[i, k, 0] < self.EPSQ:
                         ccnd[i, k, 0] = 0.0
 
-        if Model["uni_cld"]:
-            if Model["effr_in"]:
-                for k in range(LM):
-                    k1 = k + kd
-                    for i in range(IM):
-                        cldcov[i, k1] = Tbd["phy_f3d"][i, k, Model["indcld"] - 1]
-                        effrl[i, k1] = Tbd["phy_f3d"][i, k, 1]
-                        effri[i, k1] = Tbd["phy_f3d"][i, k, 2]
-                        effrr[i, k1] = Tbd["phy_f3d"][i, k, 3]
-                        effrs[i, k1] = Tbd["phy_f3d"][i, k, 4]
-            else:
-                for k in range(LM):
-                    k1 = k + kd
-                    for i in range(IM):
-                        cldcov[i, k1] = Tbd["phy_f3d"][i, k, Model["indcld"] - 1]
```

Comment:
Are you sure this line isn't used? Is it always overwritten by L754 below?

Reply:
Yes. Model["uni_cld"] is to to False and therefore everything inside this loop is not used.